lib/utils/job-utils/racadm-parser.js

```diff
@@ -2,20 +2,25 @@
 
 "use strict";
 
-var di = require('di');
+var di = require('di'),
+    xml2js = require('xml2js');
```

Comment:
xml2js library is not a newly introduced one, it is already included in on-tasks package.json.